docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1290,7 +1294,8 @@ Status Codes:
           "playdate:latest"
         ],
         "Size": 0,
-        "VirtualSize": 2429728
+        "VirtualSize": 2429728,
+        "Labels": {}
```

Comment:
Oh! hold on, looks like those are some really old images I still had in my cache, newer images *did* give me `"Labels":{}`